### ISSUE TITLE
Default Value for —count doesn’t work

### ISSUE THREAD

**Issue by Dororo02** [boxblinkracer/shopware-ai-demodata#4]
The Documentation says, that there is a defaut value for the count argument (1). But when running the ai-demodata:generate:products command without the —count argument, the following exception occours:

```
In ProductGenerator.php line 123:
                                                                                                                                                                                                                                       
  [TypeError]                                                                                                                                                                                                                          
  AIDemoData\Service\Generator\ProductGenerator::generate(): Argument #2 ($maxCount) must be of type int, null given, called in /private/var/www/shopware6/vendor/boxblinkracer/shopware-ai-demodata/src/Command/ProductGenerateCommand.php on line 155
```

**Comment by boxblinkracer** [comment]
hi

oh thank you for this
i've just published a fix,
comes with next update

**Comment by boxblinkracer** [comment]
released in 1.3.1
